.github/composite-actions/run-pg-upgrade/action.yml

```diff
@@ -27,7 +27,7 @@ runs:
         export PATH="$HOME/${{ inputs.pg_new_dir }}/bin:$HOME/${{ inputs.pg_old_dir }}/bin:$PATH"
         export LD_LIBRARY_PATH="$HOME/${{ inputs.pg_new_dir }}/lib:$HOME/${{ inputs.pg_old_dir }}/lib:$LD_LIBRARY_PATH"
         ~/${{ inputs.pg_new_dir }}/bin/pg_upgrade -U runner -b ~/${{ inputs.pg_old_dir }}/bin -B ~/${{ inputs.pg_new_dir }}/bin \
-        -d ~/${{ inputs.pg_old_dir }}/data -D ~/${{ inputs.pg_new_dir }}/data -p 5432 -P 5433 -j 4 --link --verbose
+        -d ~/${{ inputs.pg_old_dir }}/data -D ~/${{ inputs.pg_new_dir }}/data -p 5432 -P 5433 -j 4 --link --verbose --retain
```

Comment:
why this change?

Reply:
I think we can remove `--retain` option as it retains log/sql files after successful completion but we are not even uploaded these files into artifacts if pg_upgrade completes successfully.

Reply:
Without this, we cannot see pg_upgrade log when JDBC test fails after pg_upgrade succeeds.